packages/api-reference/src/components/Card/Card.vue

```diff
@@ -18,6 +18,15 @@
   );
   position: relative;
 }
+/*
+ * Don't constrain card height on mobile
+ * (or zoomed in screens)
+ */
+@media (max-width: 600px) {
```

Comment:
should we use `640px` here? ([ref](https://tailwindcss.com/docs/responsive-design))

Reply:
but that's what we used elsewhere actually i saw we updated it so nevermind!